internal/criocli/criocli.go

```diff
@@ -725,6 +728,11 @@ func getCrioFlags(defConf *libconfig.Config) []cli.Flag {
 			Usage:   fmt.Sprintf("Determines whether we pin and remove IPC, network and UTS namespaces and manage their lifecycle (default: %v)", defConf.ManageNSLifecycle),
 			EnvVars: []string{"CONTAINER_MANAGE_NS_LIFECYCLE"},
 		},
+		&cli.BoolFlag{
+			Name:    "drop-infra",
+			Usage:   fmt.Sprintf("determines whether we drop the infra container when appropriate. Requires ManageNSLifecycle to be true (default: %v)", defConf.DropInfra),
```

Comment:
> Requires ManageNSLifecycle to be true (default: %v)", defConf.DropInfra)

As a user, I would think that `(default: ...)` prints the value of `ManageNSLifecycle`, not the DropInfra.